src/extension.ts

```diff
@@ -27,9 +28,9 @@ let telemetry: ITelemetry;
 async function init(context: vscode.ExtensionContext, git: GitAPI, repository: Repository): Promise<void> {
 	Logger.appendLine('Git repository found, initializing review manager and pr tree view.');
 
-	const configuration = new VSCodeConfiguration();
-	await configuration.loadConfiguration();
-	configuration.onDidChange(async _ => {
+	Keychain.init(context);
+	await migrateConfiguration();
```

Comment:
we may want to migrate before we intialize the keychain, otherwise the very first time I use the new version, it asks me to login again.

Reply:
`migrateConfiguration` calls `Keychain.migrateToken`, which requires the keychain to be initialized.

I don't see how initializing the keychain causes us to prompt the user? We just set the default values of two locals.

Reply:
My assumption is wrong here. The extension asks me very often to sign in and it happens the first time as well when I launch the debugging instance with this change but it seems they are unrelated. We need some logging to see why that happens.